### PR TITLE
Add intro text for sub-pages

### PR DESCRIPTION
**Pull request by eparish**
Also changed index page with new ideas.

Still don't love the tagline for the best practices page, might rewrite.

@zoubeiri

### CODE REVIEW

**Comment by zoubeiri** [comment]
Looks good. Any thoughts @jmcampanini ?

File: best-practices.md [hunk 1]
```diff
@@ -1,31 +1,11 @@
...
-These are some helpful best practices.
-These are some helpful best practices.
-These are some helpful best practices.
-These are some helpful best practices.
-These are some helpful best practices.
-These are some helpful best practices.
-
-This is some great text about best practices. wow.
+The best practices described here were collected and honed by developers at Palantir.
+Whether you're just starting a new project and wondering what tooling to use, want to get better at giving code reviews, or are trying to improve your test coverage, you can benefit from others who have done it before.
```
Comment (jmcampanini) [inline best-practices.md]: long lines :-P

Comment (eparish) [inline best-practices.md]: Omg okay I can fix

**Comment by jmcampanini** [comment]
👍  looks good. would like to get some linting in here, so we have consistency with line length. but that's not a blocker.

**Pull request merged**